### PR TITLE
[le11 ] xorg-server: update to 21.1.1

### PR DESCRIPTION
**Pull request by SupervisedThinking**
Update to xorg-server 21.1.1

- bumped dependencies
- renamed / updated / dropped patches
- some clean ups in xf86-video-* packages
- switched xorg-server to meson toolchain

Announcement
https://lists.x.org/archives/xorg-announce/2021-October/003115.html
https://lists.x.org/archives/xorg-announce/2021-November/003116.html

### CODE REVIEW

**Comment by SupervisedThinking** [comment]
@heitbaum did you test the xorg server bump any further? Any flaws?

**Comment by heitbaum** [comment]
Morning - sorry for taking the time - needed to finish the 5.15.0 testing on all of mine .... So I have tested this on top of :master with #5738 and all working as expected.

```
=== tested on ===
Generic-legacy.x86_64-devel-20211103220421-7ebc4f8
Linux nuc11 5.15.0 #1 SMP Wed Nov 3 22:32:00 AEDT 2021 x86_64 GNU/Linux
Starting Kodi (20.0-ALPHA1 (19.90.101) Git:aff7e96baad0cadabb378817a078166cc2031cdf). Platform: Linux x86 64-bit
Using Release Kodi x64
Kodi compiled 2021-11-03 by GCC 10.3.0 for Linux x86 64-bit version 5.15.0 (331520)
Running on LibreELEC (heitbaum): devel-20211103220421-7ebc4f8 11.0, kernel: Linux x86 64-bit version 5.15.0
FFmpeg version/source: 4.4-Kodi
Host CPU: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz, 8 cores available
[    0.000000] DMI: Intel(R) Client Systems NUC11PAHi7/NUC11PABi7, BIOS PATGL357.0041.2021.0811.1505 08/11/2021
CApplication::CreateGUI - using the x11 windowing system
RetroPlayer[PROCESS]: Registering process control for X11
CRenderSystemGL::InitRenderSystem - Version: 4.6 (Core Profile) Mesa 21.2.5, Major: 4, Minor: 6
GL_RENDERER = Mesa Intel(R) Xe Graphics (TGL GT2)
GL_VERSION = 4.6 (Core Profile) Mesa 21.2.5
libva info: VA-API version 1.13.0
vainfo: VA-API version: 1.13 (libva 2.13.0)
vainfo: Driver version: Intel iHD driver for Intel(R) Gen Graphics - 21.4.0 (7ebc4f8ddb)
```

File: packages/x11/driver/xf86-input-libinput/package.mk [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="xf86-input-libinput"
-PKG_VERSION="1.0.1"
-PKG_SHA256="fddec49c115591918475155bf16aaf23017d7f814cee7823a0c11f867aca245b"
+PKG_VERSION="1.2.0"
```
Comment (lrusak) [inline packages/x11/driver/xf86-input-libinput/package.mk:6]: Remember that this package isn't built by default so you likely have to force building it.

**Comment by heitbaum** [comment]
@SupervisedThinking - Update 21.1.1 - https://lists.x.org/archives/xorg/2021-November/060806.html

**Comment by SupervisedThinking** [comment]
I've updated to 21.1.1 & cleaned the commit messages, I guess the PR should be fine now.

**Review by CvH — APPROVED** [review]

**Pull request merged**